src/shims/unix/fd.rs

```diff
@@ -380,6 +435,31 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         Ok((-1).into())
     }
 
+    /// Function used to update the ready list of an epoll_event. It takes in all the readiness
+    /// flags of a file description and return event as ready if any of the flag is monitored
```

Comment:
The grammar here doesn't seem right, I can't parse this sentence. Should it be "returns events as ready"?

But the function doesn't return anything...

Reply:
Updated. "return events as ready" here means insert an ``EpollReturn`` to the ready list because this is how we notify the caller that "this file description is ready" during ``epoll_wait``